protocol-designer/src/pages/Designer/ProtocolSteps/Timeline/TimelineToolbox.tsx

```diff
@@ -131,9 +131,9 @@ export function TimelineToolbox({
       maxHeight={`calc(100vh - ${NAV_BAR_HEIGHT_REM}rem - 2 * ${SPACING.spacing12})`}
       width={`${sidebarWidth / 16}rem`}
       title={
-        <Flex flexDirection={DIRECTION_COLUMN}>
+        <Flex flexDirection={DIRECTION_COLUMN} gridGap={SPACING.spacing12}>
```

Comment:
```suggestion
        <Flex flexDirection={DIRECTION_COLUMN} gridGap={SPACING.spacing8}>
```

![Screenshot 2025-06-04 at 3 15 58 PM](https://github.com/user-attachments/assets/7c3ecfff-447c-4ea9-988b-64c6eba16180)